lib/nexthop.h

```diff
@@ -83,6 +83,7 @@ struct nexthop {
 #define NEXTHOP_FLAG_MATCHED    (1 << 4) /* Already matched vs a nexthop */
 #define NEXTHOP_FLAG_FILTERED   (1 << 5) /* rmap filtered, used by static only */
 #define NEXTHOP_FLAG_DUPLICATE  (1 << 6) /* nexthop duplicates another active one */
+#define NEXTHOP_FLAG_RNH_FILTERED  (1 << 7) /* rmap filtered, used by rnh */
```

Comment:
Donald's PR #4154 is going to touch these flags values too - will you re-sync after the merge?

Reply:
Yes the looser will have to :)